mappings/net/minecraft/world/gen/feature/MegaTreeFeatureConfig.mapping

```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/class_4636 net/minecraft/world/gen/feature/MegaTreeFeatureCo
 	METHOD <init> (Lnet/minecraft/class_4651;Lnet/minecraft/class_4651;Ljava/util/List;III)V
 		ARG 5 heightInterval
 		ARG 6 crownHeight
+	METHOD method_23408 deserialize2 (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_4636;
```

Comment:
Hmm, is there a name conflict?

Reply:
Yeah, the parent class already has a `deserialize` method. Tbh, I'm not a big fan of `deserialize2`, but it's already used in BranchedTreeFeatureConfig and I don't have any other idea

Reply:
if both methods are static, then it's an enigma bug.  try rename the method in a text editor and see if travis builds.

Reply:
Yeah, I'm pretty sure it's an Enigma bug. I encountered it with #858 but forgot to make an issue.

Reply:
Done, and travis builds